### PR TITLE
introduce OwnerType

### PR DESCRIPTION
**Pull request by wxiaoguang**

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -113,6 +113,11 @@ unknown = Unknown
 
 rss_feed = RSS Feed
 
+concept_system_global = Global
+concept_person_individual = Individual
+concept_code_repository = Repository
+concept_user_organization = Organization
```
Comment (yp05327) [inline options/locale/locale_en-US.ini:119]: `individual` and `organization` are both user types. So maybe `concept_user_individual` is better?

Comment (wxiaoguang) [reply options/locale/locale_en-US.ini:119]: Sure, feel free to improve~~

File: models/types/ownertype.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package types
```
Comment (yp05327) [inline models/types/ownertype.go:1]: About the file path and the package name.
We have many `XxxType` in different files now, if we add `OwnerType` into `types` package in this PR, does this mean we should move the others into this folder/package in the future?

Comment (wxiaoguang) [reply models/types/ownertype.go:1]: I think so, if a `type` is shared by different packages.

But if a type is specialized for one model, then keep it in its own package.

Comment (yp05327) [reply models/types/ownertype.go:1]: I see. How about convert the folder name into `shared` to make it more clearly to other maintainers?

Comment (wxiaoguang) [reply models/types/ownertype.go:1]: maybe `sharedtypes`? I guess golang doesn't like underline in package name ...

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (yp05327) [reply options/locale/locale_en-US.ini:119]: Thank you for your kindness. 
I will make the change later.

File: models/types/ownertype.go (hunk 2 above)
Comment (yp05327) [reply models/types/ownertype.go:1]: Ah, I edited my comment.
Maybe `models/shared/ownertype.go` and package name `types` is good.
We can add more shared things into this folder if we have.
If there are too many shared types, we can move them into `models/shared/types/xxx.go` in the future.

**Pull request merged**